yarn.lock

```diff
@@ -2665,6 +2665,7 @@ __metadata:
   resolution: "@coderscamp/api@workspace:packages/api"
   dependencies:
     "@coderscamp/shared": 0.0.1
+    "@nestjs-architects/typed-cqrs": 0.1.1
```

Comment:
What are the pros and cons of this library? 
As far as I see, it's not so popular.

Reply:
Those are only types, so we don't need to worry. It gives us a possibility to define return types from queries and commands on a query/command level and then controls if you reaturn a correct type in a handler.